src/scheduler.cc

```diff
@@ -797,7 +797,7 @@ SchedulerThread* Scheduler::start_thread(Locker& locker) {
   if (new_thread == null) FATAL("OS thread spawn failed");
   int core = num_threads_++;
   threads_.prepend(new_thread);
-  if (!new_thread->spawn(4 * KB, core)) FATAL("OS thread spawn failed");
+  if (!new_thread->spawn(7 * KB, core)) FATAL("OS thread spawn failed");
```

Comment:
Would it be better with 8KB here, @erikcorry?

Reply:
It would be better to get it back to 4kb, just not sure how much work that would require, or if it is impossible with the espressif posix emulation

Reply:
In that case, I think we should put 8 * KB in the code.